guides/common/modules/ref_permissions-required-for-provisioning.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: REFERENCE
+
+[id="permissions-required-for-provisioning_{context}"]
+= Permissions Required for Provisioning
+
+The following table lists the permissions a non-admin user requires to provision hosts.
+
+.Permissions required for provisioning hosts
```

Comment:
The list of permissions will be different for each scenario:
* Vanilla foreman (eg: bare-metal provisioning)
* Foreman + Katello
* Satellite
* Foreman + discovery / bootdisk

Reply:
I believe only Foreman + Katello and Satellite will be applicable -- the table is to be included in sections covering KVM and KubeVirt. I added another column to provide details for permissions applicable only to Katello etc.